.pre-commit-config.yaml

```diff
@@ -0,0 +1,30 @@
+default_language_version:
```

Comment:
This file is probably OK but I feel like it's missing some configuration that we used to have for this project? How do we need those things have been accounted for? This comment mirrors a question that I asked on the sister PR of this matter

Reply:
Same answer, but for reference:
https://github.com/dbt-labs/dbt-postgres/blob/805dc3b58826e50a9c33780cbc25aa23331d27a8/pyproject.toml#L133-L157

Reply:
Per internal conversation, all linter config is now moved here.